contracts/TopShot.cdc

```diff
@@ -47,62 +47,63 @@ import FungibleToken from 0xFUNGIBLETOKENADDRESS
 
     // The network the contract is deployed on
-    pub fun Network() : String { return ${NETWORK} }
+    access(all) view fun Network() : String { return ${NETWORK} }
 
     // The address to which royalties should be deposited
-    pub fun RoyaltyAddress() : Address { return 0xTOPSHOTROYALTYADDRESS }
+    access(all) view fun RoyaltyAddress() : Address { return 0xTOPSHOTROYALTYADDRESS }
 
     // The path to the Subedition Admin resource belonging to the Account
     // which the contract is deployed on
-    pub fun SubeditionAdminStoragePath() : StoragePath { return /storage/TopShotSubeditionAdmin}
+    access(all) view fun SubeditionAdminStoragePath() : StoragePath { return /storage/TopShotSubeditionAdmin}
 
     // -----------------------------------------------------------------------
     // TopShot contract Events
     // -----------------------------------------------------------------------
 
     // Emitted when the TopShot contract is created
-    pub event ContractInitialized()
+    access(all) event ContractInitialized()
 
     // Emitted when a new Play struct is created
-    pub event PlayCreated(id: UInt32, metadata: {String:String})
+    access(all) event PlayCreated(id: UInt32, metadata: {String:String})
     // Emitted when a new series has been triggered by an admin
-    pub event NewSeriesStarted(newCurrentSeries: UInt32)
+    access(all) event NewSeriesStarted(newCurrentSeries: UInt32)
 
     // Events for Set-Related actions
     //
     // Emitted when a new Set is created
-    pub event SetCreated(setID: UInt32, series: UInt32)
+    access(all) event SetCreated(setID: UInt32, series: UInt32)
     // Emitted when a new Play is added to a Set
-    pub event PlayAddedToSet(setID: UInt32, playID: UInt32)
+    access(all) event PlayAddedToSet(setID: UInt32, playID: UInt32)
     // Emitted when a Play is retired from a Set and cannot be used to mint
-    pub event PlayRetiredFromSet(setID: UInt32, playID: UInt32, numMoments: UInt32)
+    access(all) event PlayRetiredFromSet(setID: UInt32, playID: UInt32, numMoments: UInt32)
     // Emitted when a Set is locked, meaning Plays cannot be added
-    pub event SetLocked(setID: UInt32)
+    access(all) event SetLocked(setID: UInt32)
     // Emitted when a Moment is minted from a Set
-    pub event MomentMinted(momentID: UInt64, playID: UInt32, setID: UInt32, serialNumber: UInt32, subeditionID: UInt32)
+    access(all) event MomentMinted(momentID: UInt64, playID: UInt32, setID: UInt32, serialNumber: UInt32, subeditionID: UInt32)
 
     // Events for Collection-related actions
     //
     // Emitted when a moment is withdrawn from a Collection
-    pub event Withdraw(id: UInt64, from: Address?)
+    access(all) event Withdraw(id: UInt64, from: Address?)
     // Emitted when a moment is deposited into a Collection
-    pub event Deposit(id: UInt64, to: Address?)
+    access(all) event Deposit(id: UInt64, to: Address?)
```

Comment:
I would recommend removing these events in favor of [the default events](https://github.com/onflow/flow-nft/blob/standard-v2/contracts/NonFungibleToken.cdc#L72-L86) because the default events have a lot more metadata and they will be emitted anyway